README.md

```diff
@@ -23,6 +23,8 @@ This project's aim is to build a powerful base Package Deployer template that si
     - [Import data](#Import-data)
   - [Word templates](#Word-templates)
     - [Import word templates](#Import-word-templates)
+  - [Mailboxes](#Mailboxes)
+    - [Update Approve Test and Enable shared mailboxes](#Update-Approve-Test-and-Enable-shared-mailboxes)
```

Comment:
Capitalisation should be consistent here. Matbe

`Update, approve, test, and enable shared mailboxes`